lib/content-services/search/components/search-filter/search-filter.component.ts

```diff
@@ -342,4 +356,55 @@ export class SearchFilterComponent implements OnInit, OnDestroy {
 
         return `${fieldName}:${startLimit}"${start}" TO "${end}"${endLimit}`;
     }
+
+    private getApiResponseField(context: ResultSetContext, itemType: string, fieldLabel: string): GenericFacetResponse {
```

Comment:
Rename the function as all functions are APIs
```suggestion
    private findFacet(context: ResultSetContext, itemType: string, fieldLabel: string): GenericFacetResponse {
```